src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -287,8 +287,6 @@ public virtual VerifyResult VerifyStateDependent(DataCache snapshot, Transaction
             foreach (UInt160 hash in hashes)
                 if (NativeContract.Policy.IsBlocked(snapshot, hash))
                     return VerifyResult.PolicyFail;
-            if (NativeContract.Policy.GetMaxBlockSystemFee(snapshot) < SystemFee)
```

Comment:
If this check is removed, how to stop such transactions from getting into Mempool?

Reply:
We should treat service, storage and CPU fees differently, otherwise the restrictions on fees are meaningless. This can be done later.